storage/test/storage_suite.go

```diff
@@ -6,34 +6,33 @@ import (
 	"fmt"
 	"io"
 	"io/ioutil"
-	"sort"
 
 	"gopkg.in/src-d/go-git.v4/config"
 	"gopkg.in/src-d/go-git.v4/core"
 
 	. "gopkg.in/check.v1"
 )
 
+type storer interface {
```

Comment:
This is part of the public suite test. If we make reference here to Storer at the main package we will get a cyclic dependency